src/modules/todo-list.js

```diff
@@ -1,43 +1,82 @@
-// class UI
-class UI {
-  static displayTodos=() => {
-    const toDos = [
-      {
-        index: 1,
-        description: 'Wash the dishes',
-        completed: false,
-      },
-      {
-        index: 2,
-        description: 'Complete todo list project',
-        completed: false,
-      },
-      {
-        index: 3,
-        description: 'Have fun',
-        completed: false,
-      },
-    ];
-    toDos.forEach((todo) => {
-      UI.addTodoToList(todo);
+import ToDo from './todo.js';
+
+export default class ToDoList {
+  constructor() {
+    this.toDos = [];
```

Comment:
Please ensure that the property index is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should be equal to 5) according to the project requirement. At the moment the `first` element in your array has an `index of 0`. Kindly change that so the first `element` in the array will have an `index of 1`.